packages/react/src/components/spinner/spinner.tsx

```diff
@@ -0,0 +1,45 @@
+import { c, classy, m, SpinnerProps as BaseProps } from '@onfido/castor';
+import React from 'react';
+
+export const Spinner = ({
+  size = 'large',
+  children,
+  className,
+  ...restProps
+}: SpinnerProps): JSX.Element => (
+  <LabelWrapper>
+    {{
+      children,
+      element: (
+        <div
+          {...restProps}
+          className={classy(c('spinner'), m(size), className)}
+        />
+      ),
+    }}
+  </LabelWrapper>
+);
+
+export type SpinnerProps = BaseProps & DivElementProps;
+
+const LabelWrapper = ({
+  children: { element, children },
+}: LabelWrapperProps): JSX.Element => {
+  if (!children) return element;
+
+  return (
+    <div className={classy(c('spinner-container'))}>
+      {element}
+      {children && <div className={classy(c('spinner-label'))}>{children}</div>}
```

Comment:
This check is not needed, right?
`if (!children)` above already covers that branch.

```suggestion
      <div className={classy(c('spinner-label'))}>{children}</div>
```